pkg/vpcmodel/grouping.go

```diff
@@ -8,6 +8,7 @@ import (
 )
 
 const commaSepartor = ","
+const implies = " => "
```

Comment:
"implies" might be confusing here. I would use "connectsTo" or simply "arrow"

Reply:
done